cmake/OpenSimConfig.cmake.in

```diff
@@ -54,3 +64,14 @@ include("@PACKAGE_OPENSIM_INSTALL_CMAKEDIR@/@CMAKE_PROJECT_NAME@Targets.cmake")
 list(APPEND @CMAKE_PROJECT_NAME@_LIBRARIES osimTools)
 
 check_required_components(OpenSim)
+
+# Python/Java Wrapping
+#---------------------
+set(@CMAKE_PROJECT_NAME@_PYTHON_WRAPPING @BUILD_PYTHON_WRAPPING@)
+
+set(@CMAKE_PROJECT_NAME@_JAVA_WRAPPING @BUILD_JAVA_WRAPPING@)
+set(@CMAKE_PROJECT_NAME@_JAVA_FILES_DIR 
+   @CMAKE_INSTALL_PREFIX@/@OPENSIM_INSTALL_JAVAJARDIR@/org)
+set(@CMAKE_PROJECT_NAME@_JAR_FILE 
+    @CMAKE_INSTALL_PREFIX@/@OPENSIM_INSTALL_JAVAJARDIR@/@SWIG_JAVA_JAR_NAME@)
```

Comment:
To make the OpenSim installation relocatable, you cannot use `@CMAKE_INSTALL_PREFIX@` in the Config file. Look at how the other variables that contain install prefix are handled (e.g., `PACKAGE_OPENSIM_INSTALL_CMAKEDIR`). For documentation on how to do this, see https://github.com/opensim-org/opensim-core/blob/master/cmake/CMakeLists.txt#L14 and https://cmake.org/cmake/help/v3.0/module/CMakePackageConfigHelpers.html.

Basically, change these lines to use `@PACKAGE_OPENSIM_INSTALL_JAVAJARDIR@` and put `OPENSIM_INSTALL_JAVAJARDIR` in `cmake/CMakeLists.txt` as one of the `PATH_VARS` to `configure_package_config_file()`.